geode-core/src/integrationTest/java/org/apache/geode/distributed/internal/membership/adapter/GMSMembershipManagerJUnitTest.java

```diff
@@ -157,51 +163,48 @@ public void initMocks() throws Exception {
     mockMembers = new InternalDistributedMember[5];
     for (int i = 0; i < mockMembers.length; i++) {
       mockMembers[i] = new InternalDistributedMember("localhost", 8888 + i);
-      GMSMember m = (GMSMember) mockMembers[i].getNetMember();
-      UUID uuid = new UUID(r.nextLong(), r.nextLong());
+      m = ((GMSMemberAdapter) mockMembers[i].getNetMember()).getGmsMember();
+      uuid = new UUID(r.nextLong(), r.nextLong());
       m.setUUID(uuid);
     }
     members = new ArrayList<>(Arrays.asList(mockMembers));
 
     listener = mock(DistributedMembershipListener.class);
 
     manager = new GMSMembershipManager(listener);
-    manager.init(services);
-    when(services.getManager()).thenReturn(manager);
+    manager.getGMSManager().init(services);
+    when(services.getManager()).thenReturn(manager.getGMSManager());
   }
 
   @After
   public void tearDown() throws Exception {
     if (manager != null) {
-      manager.stop();
-      manager.stopped();
+      manager.getGMSManager().stop();
+      manager.getGMSManager().stopped();
     }
   }
 
-  @Test
```

Comment:
is this test case still valid?

Reply:
yes, that test is still valid.  I don't recall why I removed it but I'll put it back.

```
@Test
  public void testSendMessage() throws Exception {
    HighPriorityAckedMessage m = new HighPriorityAckedMessage();
    m.setRecipient(mockMembers[0]);
    manager.getGMSManager().start();
    manager.getGMSManager().started();
    GMSMember myGMSMemberId = ((GMSMemberAdapter)myMemberId.getNetMember()).getGmsMember();
    List<GMSMember> gmsMembers = members.stream().map(x -> ((GMSMemberAdapter)x.getNetMember()).getGmsMember()).collect(
        Collectors.toList());
    manager.getGMSManager().installView(new GMSMembershipView(myGMSMemberId, 1, gmsMembers));
    Set<InternalDistributedMember> failures = manager.send(m.getRecipients(), m, this.services.getStatistics());
    verify(messenger).send(isA(GMSMessageAdapter.class));
    if (failures != null) {
      assertEquals(0, failures.size());
    }
  }
```